x-pack/legacy/plugins/uptime/server/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -33,22 +29,30 @@ export class UMKibanaBackendFrameworkAdapter implements UMBackendFrameworkAdapte
       case 'POST':
         this.server.route.post(routeDefinition, handler);
         break;
+      case 'POST':
+        this.server.route.post(
+          {
+            path,
+            validate,
+          },
+          handler
+        );
+        break;
+      case 'POST':
+        this.server.route.post(
+          {
+            path,
+            validate,
+          },
+          handler
+        );
+        break;
```

Comment:
3  'POST' cases?

Reply:
yikes... this is the result of a mistake while rebase/merging. Thanks!

Reply:
Fixed in https://github.com/elastic/kibana/pull/51403/commits/d7601be47cc7e0d74c107f5072965e850bece977.